sherpa/astro/io/tests/test_io_response.py

```diff
@@ -245,14 +238,14 @@ def test_write_arf_ascii(make_data_path, tmp_path):
 @requires_data
 @requires_fits
 def test_write_arf_fits_chandra_acis_hetg(make_data_path, tmp_path):
-    """Check we can write out an ARF with BIN_LO/HI as a FITS file.
+    """Check we can write out a Chandra ACIS grating ARF as a FITS file.
+
+    This used to write out BIN_LO/HI values too, until #1564 was fixed.
     """
```

Comment:
No need to dwell on the past. I cna use "git blame" if you want to find out who changed it last and other git tools if I want the full history. In this particular case, I don't think there is much benefit of knowing what it used to do since the bin_lo/hi are removed cleanly. Let's keep comments like this for cases where the history matters for the current design.
```suggestion
    """Check we can write out a Chandra ACIS grating ARF as a FITS file.
    """
```